grafana/resource_user.go

```diff
@@ -101,3 +106,25 @@ func DeleteUser(d *schema.ResourceData, meta interface{}) error {
 	}
 	return client.DeleteUser(id)
 }
+
+func ExistsUser(d *schema.ResourceData, meta interface{}) (bool, error) {
+	client := meta.(*gapi.Client)
+	userId, _ := strconv.ParseInt(d.Id(), 10, 64)
+	_, err := client.User(userId)
```

Comment:
Took a look at the Grafana client to see if we could compare against specific UserNotFound errors, but it it seems to be a simple HTTP wrapper, so this is fine 👍